demo-app/src/App.js

```diff
@@ -0,0 +1,38 @@
+import './App.css';
+import Counter from './components/counter/Counter';
+import SearchForm from './components/search-form/SearchFrom';
+import GenreSelect from './components/genre-select/GenreSelect';
+import React, { useState } from 'react';
+
+
+function App() {
+
+  const [searchResults, setSearchResults] = useState([]);
```

Comment:
In my opinion here searchInput or searchValue. Because 2you use this state below to safe string.
```suggestion
  const [searchResults, setSearchResults] = useState('Titanic');
```